run_interactive.py

```diff
@@ -158,20 +158,15 @@ def main(argv):
       decode_state, result_tokens = engine.generate(params, decode_state)
       result_tokens = result_tokens.convert_to_numpy()
       res = result_tokens.get_result_at_slot(slot)
-      stop_tokens = set(tokenizer.tokenizer.stop_tokens)
+      stop_tokens = set(tokenizer.stop_tokens)
```

Comment:
JetStream didn't implement the stop_tokens for SentencePiece. You might need to add that function to JetStream.

Reply:
I did: https://github.com/google/JetStream/pull/84